platform_api/orchestrator/job.py

```diff
@@ -26,7 +28,7 @@
 # `{job-name}{JOB_USER_NAMES_SEPARATOR}{job-project-name}.jobs.neu.ro`.
```

Comment:
minor: we could update those comments too (here and in platform_api/handlers/validators.py)